proto/raft_serverpb.proto

```diff
@@ -103,6 +103,8 @@ message RegionLocalState {
     PeerState state = 1;
     metapb.Region region = 2;
     MergeState merge_state = 3;
+    // The apply index corresponding to the storage when it's initialized.
```

Comment:
apply index？

Reply:
Yes. tablet index is actually (max applied) index.

Reply:
how about describing the tablet naming rule here

Reply:
It's TiKV's implementation details, I would like to make it inside TiKV.